### ISSUE TITLE
Why does this repository exist?

### ISSUE THREAD

**Issue by JorisGoosen** [jasp-stats/JASP-modules-backup#3]
@boutinb @timdraws @vandenman maybe you guys know why?

Otherwise I can just remove this right? There is already a JASP-modules which is a fullblown repository... So why we would need a backup is beyond me...

**Comment by vandenman** [comment]
No idea. Maybe @AlexanderLyNL knows? There was an idea to keep the module development private, but that didn't work out.

**Comment by AlexanderLyNL** [comment]
Yes there was an issue with Tim Draw's branch. I'd like to delete this once Tim's work is merged.

**Comment by JorisGoosen** [comment]
So who is merging this? As in, is active work being done on this?

**Comment by AlexanderLyNL** [comment]
@timdraws

**Comment by timdraws** [comment]
I am working on another branch, this one can be deleted. We are pushing all ML things to Koen's repository now.
